bundles/SeoBundle/src/Model/Redirect.php

```diff
@@ -134,11 +134,28 @@ public function getSource(): ?string
         return $this->source;
     }
 
+    /**
+     * Target as string, can be target path or document id
+     */
     public function getTarget(): ?string
     {
         return $this->target;
     }
 
+    /**
+     * resolved target path with handling for document ids as `target`
+     * slash ensured at the beginning of the string
+     */
+    public function getTargetPath(): string
+    {
+        $resolvedPath = (Pimcore\Model\Document::getById($this->target)?->getFullPath() ?? $this->target) ?? "";
```

Comment:
Readability is not the best. Could we maybe format this better, to make it easier to understand. 

Also you can Import the Document as a whole instead of using the FQNS.

Reply:
Readability should be improved and I improved the documentation as well.